querybook/server/models/metastore.py

```diff
@@ -298,6 +298,10 @@ class DataTableColumn(TruncateString("name", "type", "comment"), Base):
         sql.Integer, sql.ForeignKey("data_table.id", ondelete="CASCADE")
     )
 
+    data_elements = relationship(
```

Comment:
did you test this would work for both key and value association?

Reply:
yep, I tried it on a map type.